src/test/java/com/artipie/asto/RxFileTest.java

```diff
@@ -46,10 +47,11 @@
     @Test
     @Timeout(1)
     public void rxFileFlowWorks(@TempDir final Path tmp) throws IOException {
+        final Vertx vertx = Vertx.vertx();
```

Comment:
@Sammers21 `Vertx` instance is created and closed in every test of this test class. I think it will be better to manage it in `@BeforeEach`, `@AfterEach` methods